Update tutorial-create-vmss.md

boykoc opened the pull request:
Update wording to make it read better.

It might be a small change but I had to re-read 
this sentence multiple times to get it. The way it 
was written it seems to indicate that the rules
trigger things but its the metrics that trigger the 
rules.

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

boykoc committed:
Update tutorial-create-vmss.md

Update wording to make it read better.

It might be a small change but I had to re-read 
this sentence multiple times to get it. The way it 
was written it seems to indicate that the rules
trigger things but its the metrics that trigger the 
rules.
